.github/workflows/lint-tests.yml

```diff
@@ -0,0 +1,32 @@
+name: Run linters on Kedro
+
+on:
+  workflow_call:
+    inputs:
+      os:
+        type: string
+      python-version:
+        type: string
+
+jobs:
+  lint:
+    runs-on: ${{ inputs.os }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up Python 3.8
```

Comment:
Maybe a nice opportunity to bump the python version here and just run it on 3.10, since that's the most recent version we support.

Reply:
Done + renamed the tests